### ISSUE TITLE
message selectable?

### ISSUE THREAD

**Issue by misterhsp** [LithApp/Lith#175]
Will it be possible in the future for the message text to be selectable?

Thanks...

**Comment by MartinBriza** [comment]
Yeah, I'm planning to add a more desktop-like way to select text across multiple messages but it's not trivial with QML... definitely something I'd like to add though. 

Or are you asking about a more flexible way to select text in the touch-friendly copy dialog we have now?

**Comment by misterhsp** [comment]
select for copy and paste as an example.


Am 08.06.24 um 19:51 schrieb Martin Bříza:

**Comment by MartinBriza** [comment]
By long-pressing (or right-clicking) a message, you will be presented with a dialog that will allow you to copy the message. This is very rudimentary but for copying single messages it should be fine for the time being.
